### ISSUE TITLE
ormconfig is not setting default schema for oracle database

### ISSUE THREAD

**Issue by lipeavelar** [typeorm/typeorm#6272]
**Issue type:**

[ ] question
[ X ] bug report
[ ] feature request
[ ] documentation issue

**Database system/driver:**

[ ] `cordova`
[ ] `mongodb`
[ ] `mssql`
[ ] `mysql` / `mariadb`
[ X ] `oracle`
[ ] `postgres`
[ ] `cockroachdb`
[ ] `sqlite`
[ ] `sqljs`
[ ] `react-native`
[ ] `expo`

**TypeORM version:**

[ ] `latest`
[ ] `@next`
[ X ] `0.2.25` (or put your version here)

**Steps to reproduce or a small repository showing the problem:**

I have  set schema option on ormconfig.js to my schema, but, when I try to find an entity, it returns an error saying table or view does not exists, but if I use my schema explicit on entity declaration, e.g. `@Entity('MYSCHEMA.MYTABLE')`, it works fine.

Any sugestions what could I do to solve this problem?

**Comment by imnotjames** [comment]
According to comments in the oracle driver:

```
* Build full table name with database name, schema name and table name.
* Oracle does not support table schemas. One user can have only one schema.
```

Is this not the case?  Can you find where this is in the documentation somewhere?

**Comment by lipeavelar** [comment]
Issue solved by #6673
